templates/signup.html

```diff
@@ -0,0 +1,45 @@
+{% import "bootstrap/wtf.html" as wtf %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Image Analysis System</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.7/dist/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    <style type="text/css">
+        .wrapper {
+            margin: 150px;
+        }
+        .form-singin {
+            max-width: 380px;
+            margin: 0 auto;
+            padding: 15px 40px 50px;
+        }
+    </style>
+
+</head>
+<body>
+    <div class="wrapper">
+        <form class="form-signin" method="POST" action="/signup">
+            <h2 class="form-signin-heading text-center">Image Analysis System</h2>
+            <div class="form-group">
+            {{ form.hidden_tag() }}
+            {{ wtf.form_field(form.name) }}
+            {{ wtf.form_field(form.email) }}
+            {{ wtf.form_field(form.password) }}
+            </div>
+            <div class="form-group">
+            </div>
+            <div class="col text-center">
+                <button type="submit" class="btn btn-primary" >Create account</button>
+                <a href="/login" class="btn btn-secondary" role="button">Go back</a>
+            </div>
```

Comment:
This is the bootstrap form that holds the signup form. This form is submitted when the submit button is clicked.